server/model/status_page.js

```diff
@@ -53,6 +54,15 @@ class StatusPage extends BeanModel {
 
         const head = $("head");
 
+        await StatusPage.getStatusPageData(statusPage).then( (page) => {
```

Comment:
I think this call is not necessary, because the `statusPage` is representing the data row from database. I think you can get the UA from `statusPage.google_analytics_tag_id` directly.

Reply:
Looks like you are correct! Updating the PR now! :)